misc/copr_new_packages.py

```diff
@@ -33,36 +34,44 @@ def pick_project_candidates(client, projects, since):
     Magazine article). By such projects we consider those with at least one
     succeeded build and at least some project information filled.
     """
+    rawhide_pks_resp = subprocess.run(
+        ["dnf", "repoquery", "rawhide", "--queryformat", "%{name}", "*"],
```

Comment:
The problem with 

    dnf repoquery rawhide --queryformat "%{name}" "*"

in comparison to

    fedora-repoquery rawhide "*"

is that `dnf repoquery` doesn't only look into Fedora repositories but to all enabled repositories on the system (e.g. all my enabled Copr projects)

```
$ dnf repoquery rawhide --queryformat "%{name}" "*" |grep nix-singleuser
nix-singleuser
$ fedora-repoquery rawhide "*" |grep nix-singleuser
$ dnf info nix-singleuser
Available Packages
Name         : nix-singleuser
...
Repository   : copr:copr.fedorainfracloud.org:petersen:nix
...
```

Reply:
fixed, ptal